src/components/uv-scroll.js

```diff
@@ -13,19 +13,6 @@ export class UVScrollSystem {
       const increment = component.data.increment;
       map.offset.x = increment.x ? offset.x - (offset.x % increment.x) : offset.x;
       map.offset.y = increment.y ? offset.y - (offset.y % increment.y) : offset.y;
-
```

Comment:
Yay, this was super ugly, glad to see it go away. Worth just quickly double checking it still works in medium and low quality modes (it should since these are now all just using default ThreeJS materials which is great).